spec/pubid_itu/identifier_spec.rb

```diff
@@ -203,6 +203,26 @@ module Pubid::Itu
       it_behaves_like "converts pubid to pubid"
     end
 
+    context "Supplements" do
+      context "ITU-T H Suppl. 1" do
+        let(:pubid) { "ITU-T H Suppl. 1" }
```

Comment:
@mico given a pubid, how can I differentiate between a Series Supplement ("H Suppl.") vs a Document Supplement ("E.156 Suppl.")?

Reply:
@ronaldtse only by checking of number presence for now.
Now, the difference between `ITU-T H` and `ITU-T H.1` identifiers only in number presence.
Do you have ideas how we can improve it?

Reply:
Can you provide sample code to realize the check, and document in README? Thanks.

Reply:
@ronaldtse I added tests for supplements identifiers parsing and creation. Do you think it would be enough?